site/assets/js/partials/code-examples.js

```diff
@@ -20,9 +20,9 @@ export default () => {
 
   // Boosted mod: removed unused svg height, width and fill="currentColor"
   const btnHtml = [
-    '<div class="bd-code-snippet">',
-    '  <div class="bd-clipboard">',
-    '    <button type="button" class="btn-clipboard">',
+    '<div class="bd-code-snippet position-relative">',
+    '  <div class="bd-clipboard position-absolute top-0 end-0">',
+    '    <button type="button" class="btn-clipboard position-absolute end-0">',
```

Comment:
Don't we need a visually hidden label for this button ? It only contains an SVG without any text.

Reply:
There is a aria-label on the svg, I missed it at first :)